.github/workflows/github-apps-cd.yml

```
@@ -16,6 +16,9 @@ jobs:
   deploy-firebase-functions:
     needs: [github-apps-tests, firebase-functions-tests]
     runs-on: ubuntu-latest
+    env:
+      FIREBASE_PROJECT_ID: ${{ github.event_name == 'pull_request' && ${{ secrets.FIREBASE_PROJECT_ID_FOR_PREVIEW_FUNCTIONS }} || ${{ secrets.FIREBASE_PROJECT_ID }} }}
```

actionlint (1.7.7)

got unexpected character '$' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)